src/components/Filter/Filter.js

```diff
@@ -183,7 +185,27 @@ export default function Filter({
                   Clear All
                 </p>
               </Box>
-              <Box>
+              <Box
+                sx={{
+                  margin: '0px 20px'
+                }}
+              >
+                <p
+                  style={{
+                    margin: '10px',
+                    padding: '10px 20px',
+                    width: 'fit-content',
+                    position: 'relative',
+                    float: 'right',
+                    border: '1px solid #09A2E5',
+                    borderRadius: '8px',
+                    fontWeight: '600',
+                  }}
+                >
+                  Resources: {filteredResources.length}
+                </p>
+              </Box>
+              {/* <Box>
```

Comment:
Let's remove commented-out code unless we plan on bringing it back

Reply:
fixed